sirepo/status.py

```diff
@@ -99,3 +101,11 @@ def _run_tests():
             uri_router.call_api('runCancel', data=d)
         except Exception:
             pass
+
+
+def _validate_auth_state():
+    r = pkcompat.from_bytes(uri_router.call_api('authState').data)
+    m = re.search(r'SIREPO.authState = (.*?);', r)
+    assert m, f'no authState in response: {r}'
```

Comment:
Here's where things get tricky and we want to avoid logging what is returned by the server with an fstring. We'll want to truncate it with pkdformat, I think, e.g. `assert m, pkdformat('no authState in response={}', r)`

Reply:
8e5eee4e